Cargo.lock

```diff
@@ -4126,7 +4145,7 @@ checksum = "9fb5b7c98597bf41d1503ca4039be5445fd02e7aa381ae520d1c78a8a370f7f5"
 dependencies = [
  "ahash",
  "base64 0.13.0",
- "dashmap",
+ "dashmap 4.0.2",
```

Comment:
This PR introduces a lot of duplicated dependencies with multiple versions. I suggest to hold off on landing it before we can deduplicate them